### ISSUE TITLE
Fix Jetsons $metadata

### ISSUE THREAD

**Issue by ralfhandl** [oasis-open/odata-rapid#339]
Quirks of https://jetsons.azurewebsites.net/

- [ ] `"@context": "https://jetsons.rapid-pro.org/$metadata"` - should be `https://jetsons.azurewebsites.net/$metadata`
- [ ] `https://jetsons.azurewebsites.net/$metadata` should return CSDL JSON by default
- [ ] `https://jetsons.azurewebsites.net/$metadata` should support content negotiation - currently only returns JSON for `Accept: application/json`. Allowing other formats with lower weight will default to XML

**Comment by mikepizzo** [comment]
Working on a PR to fix the @context.

For content negotiation, the following was the old behavior:
No Accept => application/json
Accept: application/json => application/json
Accept: application/xml => application/xml
Accept: application/xml;q=0.8,application/json;q=0.5 => application/xml
Accept: application/xml;q=0.5,application/json;q=0.8 => application/json
Accept: application/text;q=0.1 => application/xml
Accept: application/text => application/xml

My PR currently changes the last two to return application/json if neither application/xml nor application/json is specified. Technically, these  SHOULD return 406 Not Acceptable, but that's just a *SHOULD* in http.

Todo:
Accept: application/* should prefer application/json over application/xml

**Comment by ralfhandl** [comment]
Current behavior:
- `Accept: application/json,*/*;q=0.9` => XML

In my understanding this should return JSON

**Comment by mikepizzo** [comment]
Yes; wildcards that match both xml and json currently default to xml. looking to see if I have an easy fix for that.
